.env.base

```diff
@@ -81,7 +81,9 @@ MAIL_HOST=smtp.mailtrap.io
 MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
-MAIL_SCHEME=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=noreply@restarters.test
```

Comment:
Just curious, is this a real address?

Reply:
Nah we're good. The `.test` domains are reserved https://en.wikipedia.org/wiki/.test